app/helpers.js

```diff
@@ -68,7 +68,7 @@ function updateReportCounts() {
   });
 }
 
-setInterval(updateReportCounts, 1000 * 60 * 60);
+setInterval(updateReportCounts, 1000 * 60 * 60).unref();
```

Comment:
By default, any setTimeout or setInterval timers will keep the node process running. Calling unref on a timer means that particular timer won't keep the process running if the event loop dries up otherwise. This was a real bugbear to figure out. I assume mocha must call process.exit once all the tests have finished.